akka-parsing/build.sbt

```diff
@@ -14,3 +13,4 @@ scalacOptions += "-language:_"
 sources in doc in Compile := List()
 
 enablePlugins(ScaladocNoVerificationOfDiagrams)
+disablePlugins(MimaPlugin)
```

Comment:
Hm, did we warn that parsing is not included in BC guarantees? Technically it's internal but in practice people could touch it AFAIR. (we should put up a note it's not to be used by anyone)